src/libs/actions/Policy/Category.ts

```diff
@@ -1182,8 +1182,10 @@ function setPolicyCategoryApprover(policyID: string, categoryName: string, appro
                 value: {
                     rules: {
                         approvalRules: updatedApprovalRules,
-                        pendingFields: {
-                            approvalRules: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                    },
+                    pendingRulesUpdates: {
+                        [categoryName]: {
```

Comment:
Let's include `field` name too along with the `categoryName` to ensure uniqueness. Otherwise, this can clash with `tag` names

Reply:
nice catch, I've added a utility function for that